libs/elements/layout/src/lib/components/elewa-hero/elewa-hero.component.scss

```diff
@@ -0,0 +1,102 @@
+.elewa-hero {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    align-items: left;
+    justify-content: left;
+    height: 100vh;
+       
+}
+    
+    .elewa-hero-content {    
+    color: var(--text-color);
+    position: absolute;
+    width: 50%;
+    align-items: left;
+    margin: 16px 0;
+    }
+
+    .subtitle {
```

Comment:
format this file properly, just right click and select format document using vscode default features, spaces 2.